core/src/no/group15/playmagic/ui/views/menulists/MainMenuList.kt

```diff
@@ -0,0 +1,103 @@
+package no.group15.playmagic.ui.views.menulists
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.graphics.g2d.BitmapFont
+import com.badlogic.gdx.graphics.g2d.TextureRegion
+import com.badlogic.gdx.math.Rectangle
+import com.badlogic.gdx.utils.GdxRuntimeException
+import ktx.collections.GdxArray
+import ktx.collections.gdxArrayOf
+import no.group15.playmagic.network.ClientConfig
+import no.group15.playmagic.network.NetworkContext
+import no.group15.playmagic.server.ServerConfig
+import no.group15.playmagic.ui.views.MainMenuView
+import no.group15.playmagic.ui.views.widgets.MenuItem
+import no.group15.playmagic.ui.views.widgets.MenuItemWidget
+import no.group15.playmagic.ui.views.widgets.MenuListWidget
+
+
+class MainMenuList(
+	boundingBox: Rectangle,
+	font: BitmapFont,
+	hoverBackground: TextureRegion,
+	mainMenu: MainMenuView
+) : MenuListWidget(
+	boundingBox,
+	font,
+	MenuItem.Alignment.RIGHT,
+	hoverBackground
+) {
+
+	override val itemList: GdxArray<MenuItemWidget> = gdxArrayOf()
+
+
+	init {
+		// Sandbox button
+		itemList.add(object : MenuItemWidget(
+			Rectangle(boundingBox.x, boundingBox.y + 3 * itemHeight, boundingBox.width, itemHeight),
+			font,
+			"Sandbox",
+			alignment,
+			hoverBackground
+		) {
+			override fun click(x: Float, y: Float) {
+				mainMenu.startGame(NetworkContext(
+					ClientConfig(host = "localhost"),
+					ServerConfig(host = "localhost", maxPlayers = 1)
+				))
+			}
+		})
```

Comment:
This formatting looks a bit weird to me, but if you ran the formatter in android studio and this is the result then it's fine

Reply:
No, the autoformatter puts NetworkContext on a new line, I just thought it looked cleaner without it.